pkg/mattermost/helpers.go

```diff
@@ -81,6 +81,42 @@ func setProbes(customLiveness corev1.Probe, customReadiness corev1.Probe) (*core
 		liveness.FailureThreshold = customLiveness.FailureThreshold
 	}
 
+	startUp := &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path: "/api/v4/system/ping",
+				Port: intstr.FromInt(8065),
+			},
+		},
+		InitialDelaySeconds: 1,
+		PeriodSeconds:       10,
+		FailureThreshold:    60,
+	}
+
+	if customStartup.Handler != (corev1.Handler{}) {
+		startUp.Handler = customStartup.Handler
+	}
+
+	if customStartup.InitialDelaySeconds != 0 {
+		startUp.InitialDelaySeconds = customStartup.InitialDelaySeconds
+	}
+
+	if customStartup.PeriodSeconds != 0 {
+		startUp.PeriodSeconds = customStartup.PeriodSeconds
+	}
+
+	if customStartup.FailureThreshold != 0 {
+		startUp.FailureThreshold = customStartup.FailureThreshold
+	}
+
+	if customStartup.SuccessThreshold != 0 {
+		startUp.SuccessThreshold = customStartup.SuccessThreshold
+	}
+
+	if customStartup.FailureThreshold != 0 {
+		startUp.FailureThreshold = customStartup.FailureThreshold
+	}
```

Comment:
This is the second time we check and set `customStartup.FailureThreshold` in this function. Did you mean to check something else or is this just a duplicate?

Reply:
was a copy and paster, the other have the same issue :)

Reply:
will wait for [this](https://github.com/mattermost/mattermost-operator/pull/162) to be merged to update this PR